app/exports/find_feedback_export.yml

```diff
@@ -0,0 +1,16 @@
+common_columns:
+  - email
+
+custom_columns:
+  feedback_provided_at:
+    type: string
+    format: date-time
+    example: 2020-11-01T00:00:00+00:00
+  find_url:
+    type: string
+    format: link
```

Comment:
This might be wrong. I'm just trying to get across that it links directly to the page on Find. That's covered by description though so it might be redundant. WDYT